src/extension/iSamples_resultList.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import React from "react";
+import cx from "classnames";
+import Table from './react_table';
+
+// Functional conponent to render button group
+const ButGroup = (props) => {
+  const { switchFormat, bootstrapCss, active } = props;
+
+  return (
+    <div className={cx({ "swithButtonBox": bootstrapCss })}>
+      <button className={cx({
+        "btn": bootstrapCss,
+        "btn-default": bootstrapCss,
+        "btn-sm": bootstrapCss,
+        "pull-left": bootstrapCss,
+        "margin-right-xs": bootstrapCss,
```

Comment:
What is the bootstrapCss?  Is that how you reference the css that React aggregates from all the components?

Reply:
bootstrapCss is actually a bool variable. The solr-faceted-search-react uses it to check if the CSS classes should be added to HTML elements. `cx` library uses this pattern to add CSS classes.

Reply:
Got it thank you!